### ISSUE TITLE
showall not working correctly for BigFloat

### ISSUE THREAD

**Issue by dpsanders** [dpsanders/ValidatedNumerics.jl#131]
``` julia
julia> a = DecoratedInterval(big(2), big(3), com)
[2, 3]₂₅₆

julia> showall(a)  # wrong
[2, 3]₂₅₆

julia> displaymode(format=:midpoint)  # wrong

julia> a
nothing
```

`format=:full` is "correct":

``` julia
julia> a
DecoratedInterval(Interval(2.000000000000000000000000000000000000000000000000000000000000000000000000000000, 3.000000000000000000000000000000000000000000000000000000000000000000000000000000), com)
```

However, it does not have the desired effect of being able to copy and paste this into the REPL. 
This would require the output to have `big"2.000000000000000000000000000000000000000000000000000000000000000000000000000000"`

**Comment by dpsanders** [comment]
This is mostly fixed in #146. 
Currently get 
```DecoratedInterval(Interval(2.000000000000000000000000000000000000000000000000000000000000000000000000000000, 3.000000000000000000000000000000000000000000000000000000000000000000000000000000)₂₅₆, com)
```
which is not copyable-and-pastable.

**Comment by dpsanders** [comment]
Fixed by #210.
